### DISCUSSION TITLE
SumtraPDF not working 2024.5.7

### DISCUSSION THREAD

**Discussion by MiMaxUltra** [sumatrapdfreader/sumatrapdf#4222] [Q&A]
it's not working at all this two days.....when i open it it will break... (win11, i5)  :(  anyone get an idea?

**Comment by GitHubRulesOK** [comment]
which version 
try a fresh download
 for example I  cannot use official 32bit pre-release I have to remove a section for it to work on windows 10 !

**Reply by MiMaxUltra** [reply]
Version 3.5.2, it immediately became unresponsive after opening it. It still remained the same after restarting the computer. After reinstalling the software, it returned to normal. I don’t know if the problem will occur again in the future:(

**Comment by kjk** [comment]
I tried latest pre-release 64-bit 16067 portable and it opened few documents without problem.

What is the version you're using? What exactly "it will break" means on what are the exact steps to reproduce the issue?

**Reply by MiMaxUltra** [reply]
Version 3.5.2, it immediately became unresponsive after opening it. It still remained the same after restarting the computer. After reinstalling the software, it returned to normal. I don’t know if the problem will occur again in the future:(

**Reply by MiMaxUltra** [reply]
emmm...., it happens again, idk why :(

**Reply by GitHubRulesOK** [reply]
@MiMaxUltra 
from the hints it repeats it may be your settings file is too large/corrupted ?
check if **without SumatraPDF active** rename the settings file and let SumatraPDF auto build a new settings. 
To determine WHY you need to consider if you made manual edits or if keeping 100s of files stored in it as history?
